constants/checks.py

```diff
@@ -1,76 +1,85 @@
 from collections import namedtuple
+from data.characters import Characters
 import data.event_bit as event_bit
 import data.npc_bit as npc_bit
 from event.event_reward import RewardType
 
-NameBit = namedtuple("NameBit", ["name", "bit", "reward_types"])
+NameBit = namedtuple("NameBit", ["name", "bit", "reward_types", "gate_character"])
```

Comment:
Adding gated character to each check, rather than tying it to each event module

Reply:
The reason for this was to allow 2/8 dragons to be gated behind character (Mt Zozo => Cyan, Ancient Castle => Edgar)